src/components/InfoBanner.tsx

```diff
@@ -20,47 +25,46 @@ const InfoBanner = ({
   shouldCenter,
   shouldSpaceBetween,
   title,
+  className,
+  onClick,
   ...props
 }: InfoBannerProps) => {
   const banner = (
-    <LightMode>
+    <VStack
+      className={cn(
+        "rounded-sm p-6 sm:flex-row",
+        shouldCenter ? "max-w-[55rem]" : "max-w-full",
+        isWarning ? "bg-orange-200" : "bg-indigo-100",
+        !onClick && "text-gray-600",
```

Comment:
Not sure I understand the addition of this prop and style. Why are we doing this?